docs/3_finetuner_in_action/3_2_image_to_image.md

```diff
@@ -12,10 +12,6 @@ The dataset consists of 6016 pairs of images (12032 in total).
 Totally-Looks-Like is a dataset and benchmark challenging machine-learned representations to reproduce human perception of image similarity. As shown below, each image patch in the left has a corresponding similar image patch in the right. 
 ```
 
-<p align="center">
-  <img src="https://finetuner.jina.ai/_static/ttl_overview.png" />
-</p>
-
```

Comment:
This picture wasn't displaying for some reason. Is there any way to make it appear? If we decide to remove this, you should remove the actual picture as well (`../_static/ttl_overview.png`)

Reply:
static is most about docs resources, for documentation images we put it somewhere else. I'll remove it for now since you already provide the link to the dataset